client/lxd.go

```diff
@@ -372,7 +372,7 @@ func (r *ProtocolLXD) rawWebsocket(url string) (*websocket.Conn, error) {
 	}
 
 	// Log the data
-	logger.Debugf("Connected to the websocket")
+	logger.Debugf("Connected to the websocket: %v", url)
```

Comment:
@stgraber I noticed the url has a `secret` param on it, is this bad to be logging?

Reply:
No, that's fine, it's a one time token effectively